integrations/cats/src/main/scala/sttp/tapir/integ/cats/ValidatorCats.scala

```diff
@@ -0,0 +1,12 @@
+package sttp.tapir.integ.cats
+
+import cats.Foldable
+import sttp.tapir.{ValidationError, ValidationResult, Validator}
+
+object ValidatorCats {
+  def nonEmptyFoldable[F[_]: Foldable, T]: Validator.Custom[F[T]] = Validator.Custom[F[T]]("collection != empty") {
```

Comment:
a problem with a custom validator is that it won't render in the openapi docs - while it should render as min size > 0. Wouldn't it be better to use `Validator.minSize[T, C](1)` instead?

Reply:
Totally agree! My fault 👍